Update section 'Persistence using Postgres' (Canton Installation) (#527)
digital-asset/docs.daml.com: docs/2.8.0/docs/canton/usermanual/installation.rst

----------------------------

In order to run any kind of node, you need to decide how and if you want to persist the

> data. You currently have three choices: Don't persist and just use in-memory stores which will be deleted if you restart
> your node or persist using ``Postgres`` or ``Oracle`` databases.

Reviewer: ```suggestion
data. You can choose not to persist data and instead use in-memory stores that are deleted on node restart,
 or you can persist data using ``Postgres`` or ``Oracle`` databases.
```

Reviewer: I know you were just capitalizing "Don't," but I took the opportunity to clarify the wording overall. 

Or, if you want to keep the text as is, revert the capitalization of "Don't." Our style is to use lowercase after a colon: https://developers.google.com/style/capitalization#capitalization-and-colons

> data. You currently have three choices: Don't persist and just use in-memory stores which will be deleted if you restart
> your node or persist using ``Postgres`` or ``Oracle`` databases.

Reviewer: ```suggestion
data. You can choose not to persist data and instead use in-memory stores that are deleted on node restart,
or you can persist data using ``Postgres`` or ``Oracle`` databases.
```

Reviewer: Note: I know you were just capitalizing "Don't," but I took the opportunity to clarify the wording overall.

Or, if you want to keep the text as is, revert the capitalization of "Don't." Our style is to use lowercase after a colon: https://developers.google.com/style/capitalization#capitalization-and-colons

~~~~~~~~~~~~~~~~~~~~~~~~~~

While in-memory is great for testing and demos, for more serious tasks, you need to use a database as a persistence layer.
Both the community version and the enterprise version support `Postgres <https://www.postgresql.org/>`__ as a persistence

> layer. Make sure that you have a running Postgres server and create one database per node.
> 
> Generally speaking, Canton is tested on
> `currently supported Postgres versions <https://www.postgresql.org/support/versioning/>`__ only.
> Please refer to the `Canton's release notes <https://github.com/digital-asset/canton/releases>`__ for the specific
> Postgres version which has been used to test a particular Canton release.

Reviewer: ```suggestion
layer. Make sure you have a running Postgres server and create one database per node.

Generally speaking, Canton is tested on
`currently supported Postgres versions <https://www.postgresql.org/support/versioning/>`__ only.
Check the `Canton release notes <https://github.com/digital-asset/canton/releases>`__ for the specific
Postgres version used to test a particular Canton release.
```

> Both the community version and the enterprise version support `Postgres <https://www.postgresql.org/>`__ as a persistence
> layer. Make sure that you have a running Postgres server and create one database per node.
> 
> Generally speaking, Canton is tested on
> `currently supported Postgres versions <https://www.postgresql.org/support/versioning/>`__ only.
> Please refer to the `Canton's release notes <https://github.com/digital-asset/canton/releases>`__ for the specific
> Postgres version which has been used to test a particular Canton release.

Reviewer: ```suggestion
Both the community version and the enterprise version support `Postgres <https://www.postgresql.org/>`__ as a persistence
layer. Make sure you have a running Postgres server and create one database per node.

Canton is tested on
`currently supported Postgres versions <https://www.postgresql.org/support/versioning/>`__ only.
See the `Canton release notes <https://github.com/digital-asset/canton/releases>`__ for the specific
Postgres version used to test a particular Canton release.
```

Author: I intentionally used `Generally speaking,` to soften the statement because Canton is not necessarily tested on all Postgres DB versions supported by Postgres.

For example, Canton is currently tested on Postgres 11, 12, 13, 14, 15. Postgres 11 becomes EOL (end-of-life, unsupported) on November 9th 2023 (tomorrow). The latest currently supported Postgres version is 16, but we don't test it yet as it has just been released recently.

Author: @marciw-da I changed the text, please have look.

The Postgres storage mixin is provided by the file ``storage/postgres.conf``.

If you just want to experiment, you can use `Docker <https://www.docker.com/>`__ to get a Postgres database up and

> running quickly. You may want to choose a  `Docker Postgres image <https://hub.docker.com/_/postgres/>`__ which matches
> the `Canton tested Postgres version <https://github.com/digital-asset/canton/releases>`__ for best results.

Reviewer: ```suggestion
running quickly. For best results, choose a `Docker Postgres image <https://hub.docker.com/_/postgres/>`__ that matches
the `tested Postgres version <https://github.com/digital-asset/canton/releases>`__.
```

> running quickly. You may want to choose a  `Docker Postgres image <https://hub.docker.com/_/postgres/>`__ which matches
> the `Canton tested Postgres version <https://github.com/digital-asset/canton/releases>`__ for best results.

Reviewer: ```suggestion
running quickly. For best results, choose a  `Docker Postgres image <https://hub.docker.com/_/postgres/>`__ that matches
the `tested Postgres version <https://github.com/digital-asset/canton/releases>`__.
```
